app/views/quotes/new.html.erb

```diff
@@ -17,24 +18,91 @@
+               <hr>
                <form action="" id="SubmitQuote">
+                  <div><b>Step 2:</b> Select service area</div>
+                  <div class="hidden" id="step2"> <!-- Displays instructions and gif -->
+                     <p>Identify your driveway, front door, and any area that needs to be cleared of snow and salted</p>
+                     <div>
+                        <%= image_tag('SelectArea.gif', alt: 'Select area example animation') %>
+                     </div>
+                  </div>
                   <div class="hidden row" id="displayTotals">
-                     <div class="col-12 mt-4">
+                     <div class="col-12 mt-4"> <!-- Displays total square footage selected -->
                         <div class="card">
                            <div class="card-body text-center p-1">
                               <span id="calculatedArea" class="font-weight-bold font-size-35"></span> sq ft
                            </div>
                         </div>
                      </div>
-                     <div class="col-12 mt-3">
+                     <div class="col-12 mt-3"> <!-- Displays subtotal for selected area -->
                         <div class="card">
                            <div class="card-body text-center p-1">
                               <span class="font-weight-bold currency-symbol text-secondary">$</span>
-                              <span id="totalDue" class="font-weight-bold font-size-35"></span> CAD
+                              <span id="subTotalDue" class="font-weight-bold font-size-35">subTotal</span> CAD
                            </div>
                         </div>
                      </div>
+                     <hr>
+                     <div class="col-12 mt-3"> <!-- Displays hint to add more areas -->
+                        <div>Did you know ...
+                           <p>You can select more than one area. Add side paths, backstairs, patio, etc </p>
+                        </div>
+                     </div>
+                  </div>
+                  <hr>
+                  <div><b>Step 3:</b> When would you like the service? </div>
+                  <div class="hidden row" id="displayServiceExpedition">
                      <div class="col-12 mt-3">
+                        <fieldset class="form-group">
+                           <div class="row">
+                              <div class="col-sm-10">
+                                 <% @service_expedition_time.each {|i| %>
+                                    <div class="form-check">
+                                       <input class="form-check-input" type="radio" name="serviceExpeditionCost" id="<%= i[:id] %>" value="<%= i[:price] %>" data-label="<%= i[:label] %>" checked>
+                                       <label class="form-check-label" for="<%= i[:id] %>">
+                                          <%= i[:label] %> <%= i[:id] %> - <b> $<%= i[:price] %> </b>
```

Comment:
FYI no need for spaces inside the `<b>` tags (multiple times in the PR)

Reply:
done